README.md

```diff
@@ -1,67 +1,66 @@
+| Delight users with modern UI | Access new platform features | Down-level support | 
+|:--:|:--:|:--:|
+| <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-get-started.svg" width=250 alt="Modern navigation"/></br>Powerful WinUI 3 UI | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-scenarios.svg" width=250 alt="Platform logo"/></br>Localization + power status + more</br> | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-host-integration-install-configure.svg" width=250 alt="Down-level logo"/></br>Back to Windows 10 1809 |
 
+...and more, **without having to rewrite your app!**
 
-### What is Project Reunion?
-Project Reunion is our vision for unifying and evolving the Windows developer platform to make it easier
-to build great apps that work across all the Windows 10 versions and devices people use.
+* Use your current installer *(no requirement to use MSIX, but there are [reliability/security benefits to using MSIX](https://docs.microsoft.com/windows/msix/overview#key-features)!)*
+* 100% additive *(only add what you need, leave the rest of your app as-is)*
+* Works with Win32, WPF, WinForms, and UWP apps
 
-Project Reunion makes it easier to build a great Windows app by providing a unified platform for new 
-and existing Win32 and UWP apps. It will unify access to existing Win32 and UWP APIs and make them 
-available decoupled from the OS, via tools like NuGet. 
+Plus, **it works down-level** to build 17763 (version 1809/October 2018 Update) of Windows 10!
 
-Project Reunion will help you update and modernize your existing apps with the latest functionality, 
-whether they're C++, .NET (including WPF, Windows Forms, and UWP) or React Native. As we decouple 
-existing APIs and add new APIs, we are also doing the work to polyfill, as needed, so the APIs work 
-down-level across supported versions of Windows. 
 
+### Try it now!
+[Packaged MSIX desktop apps](https://docs.microsoft.com/windows/msix/overview) can try the preview of the 0.5 release now! (link to details on how to try it)
 
-### How can I use Project Reunion?
-Check out our roadmap and what's supported with Project Reunion **[here](docs/roadmap.md)**!
+Some caveats about 0.5...
 
-We have already started with APIs like 
-**[WinUI 3](https://github.com/microsoft/microsoft-ui-xaml)**,
-**[WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/)** and 
-**[MSIX](https://docs.microsoft.com/en-us/windows/msix/overview)**
-**([MSIX-Core](https://github.com/microsoft/msix-packaging))** 
-which you can start using today!
+* Requires using MSIX *(1.0 won't require MSIX)*
+* Only includes WinUI 3 / Text rendering / Localization right now
+* WinUI 3 cannot yet be intermixed with other UI frameworks *(XAML Islands will enable this in the future)*.  To use WinUI 3, either start with a new app, or you can replace your existing app's UI completely with WinUI 3

```

Comment:
```suggestion
* WinUI 3 can't be used with other UI frameworks. (XAML Islands will enable this in the future.) To use WinUI 3, either start with a new app, or move your app's UI to WinUI XAML.
```
- Removes double space after period
- Fixes parenthetical, punctuation
- Removes unnecessary emphasis
- Optional: Changes harsh rewrite (conflicts with earlier 'no rewrite needed' message) to read more as a logical refactor step that needs to occur. Also avoids 'WinUI 3' repetition by indicating the move target is WinUI XAML. If there's another possible target, this would have to be changed.